src/OpenLoco/src/Graphics/Font.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <cstdint>
+
+namespace OpenLoco::Gfx
+{
+    enum class Font : int16_t
+    {
+        m1 = -1,
```

Comment:
Could you look at all users of m1 and m2 I've a strong feeling this isn't really a font

Reply:
It's definitely used near drawString users, however in drawString we've got code dealing with this
```cpp
if (getCurrentFont() == Font::m1)
{
    setCurrentFont(Font::medium_bold);
    _currentFontFlags = _currentFontFlags | TextDrawFlags::dark;
}
else if (getCurrentFont() == Font::m2)
{
    setCurrentFont(Font::medium_bold);
    _currentFontFlags = _currentFontFlags | TextDrawFlags::dark;
    _currentFontFlags = _currentFontFlags | TextDrawFlags::extraDark;
}
```
So m1 is bold + dark, m2 is bold + extra dark.

Reply:
Yeah but not in this PR, those special attributes should be driven by the parameters.